docs/plugins/best_practices.md

```diff
@@ -302,4 +302,75 @@ def change_style():
 get_settings().appearance.events.theme.connect(change_style)
 change_style()
 
-```
\ No newline at end of file
+```
+
+## Do not package your tests as top-level package
+
+
+```{note}
+If you are using the [napari plugin cookiecutter template](https://github.com/napari/cookiecutter-napari-plugin),
+your tests will be included within your package. That repository structure is not affected by this issue
```

Comment:
As someone really new to this, this isn't clear. I'd expect tests to be in the package, I think this means they're in the `src/plugin-name` subfolder?
Maybe can clarify this, or say something like "your tests will be by defaulted located in the proper place."

Reply:
Thanks @psobolewskiPhD! I've rephrased a bit following your advice. Let me know if this is better!

Reply:
crystal clear. love it.